packages/xstate-vue/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@xstate/vue",
+  "version": "0.0.1",
+  "description": "XState tools for Vue",
+  "keywords": [
+    "state",
+    "machine",
+    "statechart",
+    "scxml",
+    "state",
+    "graph",
+    "vue",
+    "composable"
+  ],
+  "author": "David Khourshid <davidkpiano@gmail.com>",
+  "homepage": "https://github.com/davidkpiano/xstate/tree/master/packages/xstate-vue#readme",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "sideEffects": false,
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "files": [
+    "lib/**/*.js",
+    "lib/**/*.d.ts"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/davidkpiano/xstate.git"
+  },
+  "scripts": {
+    "clean": "rm -rf dist lib tsconfig.tsbuildinfo",
+    "prebuild": "npm run clean",
+    "build": "tsc",
+    "test": "jest",
+    "prepublish": "npm run build && npm run test"
+  },
+  "bugs": {
+    "url": "https://github.com/davidkpiano/xstate/issues"
+  },
+  "peerDependencies": {
+    "@vue/composition-api": "^0.3.4",
```

Comment:
is there any estimate for vue 3 being released? once it happens we'll have to change peers here, right?

Reply:
I think it's planned for Q1 2020. It could mean changing peers if we don't want to support vue 2 anymore.